### PR TITLE
add example to threshold and intro page

### PR DESCRIPTION
**Pull request by TuringsEgo**

### CODE REVIEW

File: vision/introduction.rst [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+
+:math:`\left( \begin{array}{ccc} 39 & 230 & 0 \\ 205 & 79 & 255 \\ 15 & 114 & 165 \end{array} \right)`
+
+C++
+^^^
+
+OpenCV's C++ binding encodes images as matrices, where each element in the matrix directly encodes for the pixel value at that particular index.
+
+
+.. code-block:: ruby
```
Comment (Twinters007) [inline vision/introduction.rst]: Why a ruby block?

Comment (TuringsEgo) [inline vision/introduction.rst]: because there isn't a c++ block to my understanding

Comment (Twinters007) [inline vision/introduction.rst]: yeah there is


.. code-block:: c++

    cout <<"Hello World" << endl;

Comment (TuringsEgo) [inline vision/introduction.rst]: ...was doing C++. I'll make the appropriate changes. Let me know when you're done reading the page(s) so I can commit

File: vision/introduction.rst [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+
+   // Assignment operator
+   C = A;
+
+Java
+^^^^
+
+Java's bindings also utilize matrices to encode images, however, as is explained later on, it is a bit trickier to display images.
+
+.. code-block:: ruby
```
Comment (Twinters007) [inline vision/introduction.rst]: Same thing. Use java here

File: vision/introduction.rst (hunk 1 above)
Comment (Twinters007) [inline vision/introduction.rst]: Everything else is good

**Pull request merged**